lib/compilers/rust.ts

```diff
@@ -113,6 +113,13 @@ export class RustCompiler extends BaseCompiler {
         }
     }
 
+    override fixIncompatbleOptions(options, userOptions): any {
```

Comment:
This and base implementation should have a return type

TS is also quite stupid when it comes to overriding methods so you need to type the arguments too
```suggestion
    override fixIncompatbleOptions(options: string[], userOptions: string[]): string[] {
```